model/user_test.go

```diff
@@ -126,11 +126,11 @@ func TestUserIsValid(t *testing.T) {
 	require.True(t, HasExpectedUserIsValidError(err, "nickname", user.Id), "expected user is valid error: %s", err.Error())
 
 	user.Nickname = strings.Repeat("a", 64)
-	require.Error(t, user.IsValid())
+	require.Nil(t, user.IsValid())
```

Comment:
This was false-passing before, as `Error`, but now that is checking for `Nil` is having the expected behavior.